examples/detection_example.cpp

```diff
@@ -0,0 +1,45 @@
+#include <iostream>
+#include <opencv2/core.hpp>
+#include <opencv2/imgcodecs.hpp>
+#include "../include/feature_lib.h"
```

Comment:
Is relative import really necessary here, since we have linked feature library to this target?

Reply:
Nope, this works without a relative path as well. And the Google cpp codestyle says to avoid Unix ../ characters. I rewrite it like #include<"file.h">